build-scripts/version.py

```diff
@@ -0,0 +1,149 @@
+"""Write src/version.h and VERSION.txt from git, if available
+
+All commands emulate the Makefile's "version" target.
+Format is: [TAG] [SHA1][-dirty]
+
+This script accepts arguments on the command line with the same format
+as environment variables:
+    VERSION=<version>
+    VERSION_STRING=<version>
```

Comment:
I think it's surprising to accept both `VERSION` and `VERSION_STRING` which are synonyms, and which get overwritten by the output of git anyway.